internal/cmd/organization/command_test.go

```diff
@@ -0,0 +1,155 @@
+package organization
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+
+	ccloudv1 "github.com/confluentinc/ccloud-sdk-go-v1-public"
+	ccloudv1mock "github.com/confluentinc/ccloud-sdk-go-v1-public/mock"
+	orgv2 "github.com/confluentinc/ccloud-sdk-go-v2/org/v2"
+	orgmock "github.com/confluentinc/ccloud-sdk-go-v2/org/v2/mock"
+
+	"github.com/confluentinc/cli/internal/pkg/ccloudv2"
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	v1 "github.com/confluentinc/cli/internal/pkg/config/v1"
+	cliMock "github.com/confluentinc/cli/mock"
+)
+
+const (
+	firstOrganizationID     = "org-resource-id"
+	firstOrganizationName   = "test-org"
+	secondOrganizationID    = "org-resource-id-2"
+	secondOrganizationName  = "test-org-2"
+	organizationNameUpdated = "test-org-updated"
+)
+
+type OrganizationTestSuite struct {
+	suite.Suite
+	conf              *v1.Config
+	accountClientMock *ccloudv1mock.AccountInterface
+	V2ClientMock      *V2ClientMock
+}
+
+type V2ClientMock struct {
+	orgClientMock *orgmock.OrganizationsOrgV2Api
+}
+
+var orgOrganization = orgv2.OrgV2Organization{
+	Id:          orgv2.PtrString(firstOrganizationID),
+	DisplayName: orgv2.PtrString(firstOrganizationName),
+}
+
+var orgOrganizationTwo = orgv2.OrgV2Organization{
+	Id:          orgv2.PtrString(secondOrganizationID),
+	DisplayName: orgv2.PtrString(secondOrganizationName),
+}
+
+var orgOrganizationUpdated = orgv2.OrgV2Organization{
+	Id:          orgv2.PtrString(firstOrganizationID),
+	DisplayName: orgv2.PtrString(organizationNameUpdated),
+}
+
+func TestOrganizationTestSuite(t *testing.T) {
```

Comment:
Sorry you put all this work into this file... but we're trying to move away from this style of testing. We can accomplish everything in this file with our integration testing framework, with ~10x less code.

Reply:
No worries; it wasn't all that much work for this part. I'll remove this in a later commit.